src/library/trackmodel.h

```diff
@@ -21,7 +22,8 @@ class TrackModel {
             : m_db(db),
               m_settingsNamespace(settingsNamespace),
               m_iDefaultSortColumn(-1),
-              m_eDefaultSortOrder(Qt::AscendingOrder) {
+              m_eDefaultSortOrder(Qt::AscendingOrder),
+              position(0){
```

Comment:
Nit pick (sorry): Please a a space between ) and { here and else where. 
More infos: http://www.mixxx.org/wiki/doku.php/coding_guidelines